tabular/src/autogluon/tabular/features/generators/auto_ml_pipeline.py

```diff
@@ -62,8 +62,10 @@ class AutoMLPipelineFeatureGenerator(PipelineFeatureGenerator):
     >>>
     >>> X_test_transformed = feature_generator.transform(test_data)
     """
-    def __init__(self, enable_numeric_features=True, enable_categorical_features=True, enable_datetime_features=True,
-                 enable_text_special_features=True, enable_text_ngram_features=True, vectorizer=None, **kwargs):
+    def __init__(self, enable_numeric_features=True, enable_categorical_features=True,
```

Comment:
Could you split this change into a separate PR so #751 can benefit from the change? Ideally, could you also set `enable_raw_text_feature=False` by default so existing users don't have extra object features being generated/in-memory by default (we can smartly enable it in future). Then, whenever you are testing a text model, simply specify

```
task.fit(..., feature_generator=AutoMLPipelineFeatureGenerator(enable_raw_text_feature=True))
```

Reply:
Also, could you update the name from `enable_raw_text_feature` to `enable_raw_text_features` to be inline with the other parameters?